lib/rubocop/ast/traversal.rb

```diff
@@ -72,18 +81,18 @@ def arity_check(range)
       extend CallbackCompiler
       send_code = CallbackCompiler::SEND
 
-      ### arity == 0
+      ### children == 0
```

Comment:
Could we write `0 children` or `children count == 0` (and same with subsequent comments)

Reply:
Changed, the second one looks better to me